### PR TITLE
Sign in/up translation

### LINKED ISSUE #189: Sign up translation
- [x] add translation for sign up page

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x] add the translation hook for sign up page

- [x] add Arabic translation

- [x] fix unite test according

- [x] fix the divider in sign in/up components

Closes #189

### CODE REVIEW

**Review by hiba-machfej — APPROVED** [review]

File: public/locales/ar/translation.json [hunk 1]
```diff
@@ -343,6 +343,48 @@
       "location": "٩٠٩٤ شارع باي ميادوز ، مدينة كونري ، ٣٠٠١٢"
     }
   },
+  "signIn": {
+    "header": "مرحبًا بك مجددًا",
+    "SignInWith": "سجل الدخول عبر",
```
Comment (louisrli) [inline public/locales/ar/translation.json:348]: capitalization

File: public/locales/ar/translation.json [hunk 2]
```diff
@@ -343,6 +343,48 @@
...
+    "emailAddress": "البريد الإلكتروني",
+    "password": "كلمة المرور",
+    "confirmPassword": "تأكيد كلمة المرور",
+    "signUp":"اشترك",
+    "acceptTerms": "لقد قرأت {{terms}} و{{privacy}} وأقبلها كلها.",
+    "terms": "شروط الخدمة",
+    "privacy": "سياسة الخصوصية",
+    "haveAccount": "لديك حساب؟ {{signIn}}",
+    "signIn": "سجل الدخول",
+    "or": "أو"
```
Comment (louisrli) [inline public/locales/ar/translation.json:386]: copy-pasted

File: src/components/SignIn/index.scss [hunk 3]
```diff
@@ -57,17 +57,17 @@
...
-      display: inline-block;
-      position: relative;
-      top: -0.8rem;
-      font: normal 14px/14px "Haas Grot Text R Web", "Helvetica Neue", Helvetica,
-        Arial, sans-serif;
-      padding: 0 16px;
-      background: $lightColor;
-    }
+  .divideContent {
+    content: "Or";
```
Comment (louisrli) [inline src/components/SignIn/index.scss:63]: this isn't translated

File: src/components/SignUpForm/index.jsx [hunk 4]
```diff
@@ -2,38 +2,57 @@ import React from "react";
...
-    }),
-  acceptTerms: Joi.valid("accepted").required().messages({
-    "any.only": `You must accept our terms of service and privacy policy`,
-  }),
-});
-
 export default function SignUpForm({ submit }) {
+  const { t } = useTranslation();
+
+  const schema = Joi.object({
```
Comment (louisrli) [inline src/components/SignUpForm/index.jsx:14]: what is this

**Comment by louisrli** [comment]
in general your PRs are always too big

your PR title is "sign in / up translation" but this is definitely not what this PR does. split it

File: public/locales/ar/translation.json (hunk 2 above)
Comment (Jamil-Bailony) [reply public/locales/ar/translation.json:386]: is it okay to use from sign in section?

File: src/components/SignUpForm/index.jsx (hunk 4 above)
Comment (Jamil-Bailony) [reply src/components/SignUpForm/index.jsx:14]: I moved this object into the compeonent to be able to use `t` hook.

**Comment by Jamil-Bailony** [comment]
> in general your PRs are always too big
> 
> your PR title is "sign in / up translation" but this is definitely not what this PR does. split it

I will split this PR into two PRs, one for sign in and another for sign up. I am going to close this.

**Pull request closed without merging**